Tests/SonarScanner.MSBuild.Tasks.UnitTests/GetAnalyzerSettingsTests.cs

```diff
@@ -356,7 +356,7 @@ private GetAnalyzerSettings Execute_ConfigExists(string sonarQubeVersion, string
             var config = new AnalysisConfig
             {
                 SonarQubeVersion = sonarQubeVersion,
-                SonarQubeHostUrl = "https://localhost:9000",
+                SonarQubeHostUrl = "https://localhost:9000", // This will be classified as SonarCloud with proper sonarQubeVersion
```

Comment:
I suggest  "If a particular sonarqube version is passed, this will be classified as SonarCloud"